2021-02-01 12:35:52.408746: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcudart.so.10.1'; dlerror: libcudart.so.10.1: cannot open shared object file: No such file or directory
2021-02-01 12:35:52.408773: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
### Starting rest_stateful_client.py client ###
Context window left width cw_l: 0
Context window right width cw_r: 0
Starting sequence_id: 1
Start processing:
Model name: rm_lstm4f
Reading input ark file /home/rasapala/download/rm_lstm4f/test_feat_1_10.ark
Reading scores ark file /home/rasapala/download/rm_lstm4f/test_score_1_10.ark
Adding input name Parameter
Adding output name affinetransform/Fused_Add_

	Sequence name: aem02_st0049_oct89
	Sequence size: 250
	Sequence id: 1
Server returned error: {'error': 'Could not find sequence id in expected tensor proto field uint64_val'}
